CONTRIBUTING.md

```diff
@@ -110,14 +112,11 @@ PR. That would at least get us started on creating the failing test in the code.
 
 ## Running Tests
 
-There are 3 ways you can run the tests.
-
 * via command-line by running `npm test`
 
-* in the browser by opening `SpecRunner.html`
+* in the browser by opening `test/runner.html`
```

Comment:
you'll first have to built the test assets with `gulp test-browser`  I don't know that we have an npm alias for that